Drug - Test URL for letters of A-Z list. OCEPROJECT-4935

venglisch opened the pull request:
This PR replaces PR #184.

venglisch committed:
Drug - Test URL for letters of A-Z list. OCEPROJECT-4935

venglisch committed:
Drug - Test URL for letters of A-Z list. OCEPROJECT-4935

blairlearn reviewed (changes requested):
Please replace the print lines to use the logger and modify the error to say "No results for " the letter.

SupriyaSaxena0605 committed:
Adding test case for ticket OCEPROJECT-4905 (#217)

alinai committed:
Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

venglisch committed:
GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

venglisch committed:
Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

SupriyaSaxena0605 committed:
Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

SupriyaSaxena0605 committed:
Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

venglisch committed:
Glossary - Test URL for letters of A-Z list. OCEPROJECT-4829 (#214)

* Glossary - Test URL for letters of A-Z list. OCEPROJECT-4829

* Glossary - Removing extra white space. OCEPROJECT-4829

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

* Glossary - Test URL for letters of A-Z list. OCEPROJECT-4829

* Glossary - Removing extra white space. OCEPROJECT-4829

* Glossary - Replace printing with logging. OCEPROJECT-4829

SupriyaSaxena0605 committed:
Fixed the verifySitewideSearch test for ticket OCEPROJECT-4949 (#223)

* Fixed the verifySitewideSearch test for ticket OCEPROJECT-4949

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Re-enable the test by *removing* the exclude

Don't just comment it out, that just leaves visual clutter.

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Fixed the verifySitewideSearch test for ticket OCEPROJECT-4949

* Re-enable the test by *removing* the exclude

Don't just comment it out, that just leaves visual clutter.

SupriyaSaxena0605 committed:
Oceproject 4942 (#212)

* adding tests for ticket OCEPROJECT-4942

* Implemented review comments for PR 178 for OCEPROJECT-4942

* Adding test case for ticket OCEPROJECT-4905 (#217)

* adding tests for ticket OCEPROJECT-4942

* Implemented review comments for PR 178 for OCEPROJECT-4942

* Updated as per review comments for PR212 for OCEPROJECT-4942

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

* adding tests for ticket OCEPROJECT-4942

* Implemented review comments for PR 178 for OCEPROJECT-4942

* Updated as per review comments for PR212 for OCEPROJECT-4942

venglisch committed:
GlossaryES - Test URL for letters of A-Z list. OCEPROJECT-4930 (#215)

* GlossaryES - Test URL for letters of A-Z list. OCEPROJECT-4930

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

* Glossary - Test URL for letters of A-Z list. OCEPROJECT-4829 (#214)

* Glossary - Test URL for letters of A-Z list. OCEPROJECT-4829

* Glossary - Removing extra white space. OCEPROJECT-4829

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

* Glossary - Test URL for letters of A-Z list. OCEPROJECT-4829

* Glossary - Removing extra white space. OCEPROJECT-4829

* Glossary - Replace printing with logging. OCEPROJECT-4829

* Fixed the verifySitewideSearch test for ticket OCEPROJECT-4949 (#223)

* Fixed the verifySitewideSearch test for ticket OCEPROJECT-4949

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Re-enable the test by *removing* the exclude

Don't just comment it out, that just leaves visual clutter.

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Fixed the verifySitewideSearch test for ticket OCEPROJECT-4949

* Re-enable the test by *removing* the exclude

Don't just comment it out, that just leaves visual clutter.

* Oceproject 4942 (#212)

* adding tests for ticket OCEPROJECT-4942

* Implemented review comments for PR 178 for OCEPROJECT-4942

* Adding test case for ticket OCEPROJECT-4905 (#217)

* adding tests for ticket OCEPROJECT-4942

* Implemented review comments for PR 178 for OCEPROJECT-4942

* Updated as per review comments for PR212 for OCEPROJECT-4942

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

* adding tests for ticket OCEPROJECT-4942

* Implemented review comments for PR 178 for OCEPROJECT-4942

* Updated as per review comments for PR212 for OCEPROJECT-4942

* GlossaryES - Test URL for letters of A-Z list. OCEPROJECT-4930

* GlossaryES - Replace printing with logging. OCEPROJECT-4930

alinai committed:
Added tests to verify the banner on FactSheet Listing page,OCEPROJECT-4994 (#225)

* Added tests to verify the banner on FactSheet Listing page,OCEPROJECT-4994

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Removing the step to create the page object
of FactSheetsListingPage,OCEPROJECT-4994

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

* Added tests to verify the banner on FactSheet Listing page,OCEPROJECT-4994

* Removing the step to create the page object
of FactSheetsListingPage,OCEPROJECT-4994

venglisch committed:
Drug - Test if Home page shows results for letter A. OCEPROJECT-4932 (#229)

venglisch committed:
Genetics - Test URL for letters of A-Z list. OCEPROJECT-4940 (#216)

* Genetics - Test URL for letters of A-Z list. OCEPROJECT-4940

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

* Glossary - Test URL for letters of A-Z list. OCEPROJECT-4829 (#214)

* Glossary - Test URL for letters of A-Z list. OCEPROJECT-4829

* Glossary - Removing extra white space. OCEPROJECT-4829

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

* Glossary - Test URL for letters of A-Z list. OCEPROJECT-4829

* Glossary - Removing extra white space. OCEPROJECT-4829

* Glossary - Replace printing with logging. OCEPROJECT-4829

* Fixed the verifySitewideSearch test for ticket OCEPROJECT-4949 (#223)

* Fixed the verifySitewideSearch test for ticket OCEPROJECT-4949

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Re-enable the test by *removing* the exclude

Don't just comment it out, that just leaves visual clutter.

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Fixed the verifySitewideSearch test for ticket OCEPROJECT-4949

* Re-enable the test by *removing* the exclude

Don't just comment it out, that just leaves visual clutter.

* Oceproject 4942 (#212)

* adding tests for ticket OCEPROJECT-4942

* Implemented review comments for PR 178 for OCEPROJECT-4942

* Adding test case for ticket OCEPROJECT-4905 (#217)

* adding tests for ticket OCEPROJECT-4942

* Implemented review comments for PR 178 for OCEPROJECT-4942

* Updated as per review comments for PR212 for OCEPROJECT-4942

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

* adding tests for ticket OCEPROJECT-4942

* Implemented review comments for PR 178 for OCEPROJECT-4942

* Updated as per review comments for PR212 for OCEPROJECT-4942

* GlossaryES - Test URL for letters of A-Z list. OCEPROJECT-4930 (#215)

* GlossaryES - Test URL for letters of A-Z list. OCEPROJECT-4930

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

* Glossary - Test URL for letters of A-Z list. OCEPROJECT-4829 (#214)

* Glossary - Test URL for letters of A-Z list. OCEPROJECT-4829

* Glossary - Removing extra white space. OCEPROJECT-4829

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

* Glossary - Test URL for letters of A-Z list. OCEPROJECT-4829

* Glossary - Removing extra white space. OCEPROJECT-4829

* Glossary - Replace printing with logging. OCEPROJECT-4829

* Fixed the verifySitewideSearch test for ticket OCEPROJECT-4949 (#223)

* Fixed the verifySitewideSearch test for ticket OCEPROJECT-4949

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Re-enable the test by *removing* the exclude

Don't just comment it out, that just leaves visual clutter.

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Fixed the verifySitewideSearch test for ticket OCEPROJECT-4949

* Re-enable the test by *removing* the exclude

Don't just comment it out, that just leaves visual clutter.

* Oceproject 4942 (#212)

* adding tests for ticket OCEPROJECT-4942

* Implemented review comments for PR 178 for OCEPROJECT-4942

* Adding test case for ticket OCEPROJECT-4905 (#217)

* adding tests for ticket OCEPROJECT-4942

* Implemented review comments for PR 178 for OCEPROJECT-4942

* Updated as per review comments for PR212 for OCEPROJECT-4942

* Added tests for verifying the breadcrumb on the FactSheet List Page, … (#224)

* Added tests for verifying the breadcrumb on the FactSheet List Page, OCEPROJECT-4988

* Updated the test to remove the println() calls, OCEPROJECT-4988

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929 (#209)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* Adding test case for ticket OCEPROJECT-4905 (#217)

* GlossaryES - Test if AZ letter links return a result. OCEPROJECT-4929

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4929

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939 (#210)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Genetics - Test if AZ letter links return a result. OCEPROJECT-4939

* GlossaryES - Replace printing with logging; adjust log message. OCEPROJECT-4939

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875 (#227)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Fixed the verifySearchWithinSearch test for ticket OCEPROJECT-4875

* removed exclude statement from the testNG-Common.xml for OCEPROJECT-4875

* Adding test case for OCEPROJECT-4998 (#222)

* Adding test case for OCEPROJECT-4998

* Adding test case for ticket OCEPROJECT-4905 (#217)

* Adding test case for OCEPROJECT-4998

* Updated review comments for PR222 for OCEPROJECT-4998

* adding tests for ticket OCEPROJECT-4942

* Implemented review comments for PR 178 for OCEPROJECT-4942

* Updated as per review comments for PR212 for OCEPROJECT-4942

* GlossaryES - Test URL for letters of A-Z list. OCEPROJECT-4930

* GlossaryES - Replace printing with logging. OCEPROJECT-4930

* Genetics - Test URL for letters of A-Z list. OCEPROJECT-4940

venglisch committed:
Drug - Replace printing with logging. OCEPROJECT-4935

venglisch committed:
Resolving merge conflicts. OCEPROJECT-4935

venglisch commented:
The requested changes have been added to this PR.
